### DISCUSSION TITLE
Appeal for Account Restriction Removal

### DISCUSSION THREAD

**Discussion by zobweyt** [community/community#135643] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope this message finds you well. I am writing to address a recent notification regarding my account, which has been flagged due to potential association with a U.S.-sanctioned region. I believe this may be an error, as I am not located in or a resident of any sanctioned area.

I greatly value the services provided by GitHub and rely on them for my projects. I kindly request that you review my account and consider lifting the restrictions placed on it. If there are any additional details or documentation required to support my appeal, please let me know, and I will be happy to provide them.

Thank you for your attention to this matter. I look forward to your prompt response.

Best regards,

zobweyt

**Comment by ebndev** [comment, marked as answer]
Hi @zobweyt 👋 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
